op-devstack/presets/minimal/init_test.go

```diff
@@ -0,0 +1,13 @@
+package minimal
+
+import (
+	"testing"
+
+	"github.com/ethereum-optimism/optimism/op-devstack/presets"
+)
+
+// TestMain creates the test-setups against the shared backend
+func TestMain(m *testing.M) {
+	presets.DoMain(m, presets.WithSimpleInterop())
```

Comment:
```suggestion
	presets.DoMain(m, presets.WithMinimal())
```

The test still worked because simple interop is a superset of minimal.